packages/cdk-routing-split/package.json

```diff
@@ -19,12 +19,16 @@
         "prettier:fix": "prettier --write --ignore-path .prettierignore \"**\" --ignore-unknown"
     },
     "devDependencies": {
+        "@types/aws-lambda": "~8.10.92",
+        "@types/chance": "^1.1.3",
         "@types/jest": "~27.4.0",
         "@types/mssql": "~7.1.4",
         "@types/node": "~17.0.17",
         "@typescript-eslint/eslint-plugin": "~5.11.0",
         "@typescript-eslint/parser": "~5.11.0",
         "aws-cdk-lib": "~2.9.0",
+        "aws-sdk": "^2.1078.0",
```

Comment:
this is breaking things for me. Nathan has noted in the past that this is automatically included by aws, and should not be included here. This is the error I received: https://stackoverflow.com/questions/57088357/aws-lambda-runtime-importmoduleerror-error-cannot-find-module-jmespath

Reply:
I have removed it and it works.

Reply:
@BennuAshes @srdjan-drca this should remain in dev-dependencies. Yes, aws-sdk is included on AWS' side which is why we only include it in dev-deps and not the dependencies list. The reason you'd keep it here is so that if you want to reference something from it, typescript won't complain about not having the associated types.